src/lib/evil/stat.c

```diff
@@ -0,0 +1,130 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+//#include <iostream>
+#include <Windows.h>
```

Comment:
try to use `evil_windows.h` instead, that has the proper WIN_LEAN_AND_MEAN guards to deal with winsock/winsock2 problem.